strings.js

```diff
@@ -32,6 +32,24 @@ function extractCurrencyValue(str) {
 }
 
 const money = "$340"
-
 console.log(extractCurrencyValue(money))
 
+
+let words = [];
+let wordStartIndex = 0;
```

Comment:
move these 2 variables inside of the function, it should be unique for each function call

Reply:
done)